elasticsearch/src/test/java/org/hibernate/search/elasticsearch/test/ElasticsearchSchemaNamingErrorsIT.java

```diff
@@ -267,6 +268,7 @@ public void configureFieldMetadata(String name, FieldMetadataBuilder builder) {
 	 */
 	@Entity
 	@Indexed
+	@Table(name = "ConcreteOnCompositeEntity")
```

Comment:
This is not strictly necessary, since the resulting name for this entity will be under 64 characters anyway. But I guess it won't hurt :)
On a side note, we have quite a few other nested classes in other tests, and they seem to be doing just fine without explicit table names. We've probably been lucky not to reach the 64 character limit until now.